packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -17,8 +17,8 @@ const XPATH_SELECTOR_REGEXP = [
     /^([a-z0-9|-]*)/,
     // optional . or # + class or id
     /(?:(\.|#)(-?[_a-zA-Z]+[_a-zA-Z0-9-]*))?/,
-    // optional [attribute-name="attribute-selector"]
-    /(?:\[(-?[_a-zA-Z]+[_a-zA-Z0-9-]*)(?:=(?:"|')([a-zA-z0-9\-_. ]+)(?:"|'))?\])?/,
+    // optional [attribute-name="attribute-selector"]. "\p{L}" - all letters (case-insensitive), "\p{N}" - all numbers
+    /(?:\[(-?[_a-zA-Z]+[_a-zA-Z0-9-]*)(?:=(?:"|')([\p{L}\p{N}\-_. ]+)(?:"|'))?\])?/u,
```

Comment:
I know this is not directly related to your change but it might be nice to improve this while we are at it.

If the idea of `[_a-zA-Z]+` is to match `[a-zA-Z0-9_]+` this can be simplified using `\w+`.
A similar thing goes for `[_a-zA-Z0-9-]*` with `[\w|\d]*`.

I'm curious if this will work but it's not a blocker for me and we can merge without this change 👍